app/assets/stylesheets/settings.scss

```diff
@@ -561,7 +579,7 @@
       font-size: 0.9em;
     }
     .github-repo-featured-indicator {
```

Comment:
I can't find a reference to this CSS selector anywhere in the code except for this file. Are you sure it's used?